src/libcore/option.rs

```diff
@@ -1064,6 +1064,20 @@ impl<T> From<T> for Option<T> {
     }
 }
 
+#[stable(feature = "option_ref_from_ref_option", since = "1.30.0")]
+impl<'a, T> From<&'a Option<T>> for Option<&'a T> {
+    fn from(o: &'a Option<T>) -> Option<&'a T> {
+        o.as_ref()
+    }
+}
+
+#[stable(feature = "option_ref_from_ref_option", since = "1.30.0")]
```

Comment:
Beta should have been branched off, so please change these versions to 1.31.0.

Reply:
Master has not yet been promoted to beta (we're waiting to make sure edition is settled to avoid many backports)

Reply:
Ok then.